packages/web-twig/src/Resources/components/Header/stories/HeaderInvertedWithActionsAndDialog.twig

```diff
@@ -36,6 +36,12 @@
                     Marian
                 </HeaderButton>
             </HeaderNavItem>
+            <HeaderNavItem>
+                <Button color="inverted" isSquare UNSAFE_className="ml-700">
```

Comment:
Currently, the necessary gap can only be done via an utility class:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/136f2280-dab7-43cd-99a8-7c3f20fbd2da)

## Possible Improvement

We can go even further and provide the spacing in `HeaderNavItem` automatically. But then we'd need to reset the spacing in `HeaderLink` and `HeaderButton` components so they only take as much space as their parent `HeaderNavItem`.

Shall we go for it?

Reply:
We can look into how Jobs uses it and decide, what do you think? If it won't break anything, I see it as a nice feature.

Reply:
As discussed, I simplified the structure as follows:

- `HeaderDesktopActions`
  - `HeaderLink` / `HeaderButton`
  - `Button` (or "anything" else)